files/image_config/config-setup/config-setup

```diff
@@ -99,28 +103,119 @@ run_hookdir() {
     return $exit_status
 }
 
+initalize_configdb()
```

Comment:
can instead call config reload -y instead of implementing here? it is code duplication.

Reply:
@lguohan I was thinking other way when making change. config reload/save command does stop/start services which is not needed during config-setup script. 
We can update those commands to call config-setup shell script with maintaining stop/start of services.
That way code will be present only in config-setup for config db generation.

Reply:
the reason the other way is preferred is that we can add more unit tests for the python script, but not bash script.

Reply:
@lguohan. Python script will still exist but internally will call shell script command. Here is the reference PR: https://github.com/Azure/sonic-utilities/pull/919

Reply:
@Iguohan Updated to use python commands.